src/orion/algo/ax/axoptimizer.py

```diff
@@ -112,3 +112,47 @@ def is_done(self):
         """Return True, if an algorithm holds that there can be no further improvement."""
         # NOTE: Drop if base implementation is fine.
         return super(AxOptimizer, self).is_done
+
+def orion_space_to_axoptimizer_space(orion_space):
+    """Convert Orion's definition of problem's domain to an axoptimizer
+    compatible one."""
+    axoptimizer_space = []
+
+    for key, dimension in orion_space.items():
+        dimensions_dict = {}
+        dimensions_dict["name"] = dimension.name
+
+        if (dimension.prior_name == 'choices'):
+            dimensions_dict["type"] = "choice"
+            dimensions_dict["values"] = dimension.categories
```

Comment:
```suggestion
            dimensions_dict["values"] = dimension.interval()
```
It's a bit weird but when using a transformed space we cannot access `.categories`. However `.interval()` returns the categories for a categorical dimension so that would work.